### PR TITLE
Updated verilator waivers so the sim builds

### PR DESCRIPTION
**Pull request by HU90m**

### CODE REVIEW

**Review by hcallahan-lowrisc — APPROVED** [review]

**Review by GregAC — APPROVED** [review]

**Review by marnovandermaas — COMMENTED** [review]
Thanks for making these changes. I had been meaning to fix the Verilator lint file for a while. On which version of Verilator did you test this?

File: ibex_demo_system_core.core [hunk 1]
```diff
@@ -24,7 +24,7 @@ filesets:
 
   files_lint_verilator:
     files:
-      - lint/verilator_waiver.vlt: {file_type: vlt}
+      - vendor/lowrisc_ibex/lint/verilator_waiver.vlt: {file_type: vlt}
```
Comment (nbdd0121) [inline ibex_demo_system_core.core:27]: I think this vlt file is already included by ibex core file

Comment (marnovandermaas) [reply ibex_demo_system_core.core:27]: Did you mean to add `demo_system_verilator_lint.vlt` here?

Comment (HU90m) [reply ibex_demo_system_core.core]: Good point. Removed this file.

**Comment by HU90m** [comment]
> Thanks for making these changes. I had been meaning to fix the Verilator lint file for a while. On which version of Verilator did you test this?

Verilator 4.210 (The officially supported ibex version)

**Comment by HU90m** [comment]
> I think the lint that you're trying to mute is a legit warning.
> 
> In ibex_register_file_fpga, if RdataMuxCheck is not enabled then the oh_raddr_a_err and oh_raddr_b_err wires are indeed not driven. Probably that should be a RTL fix instead of lint fix.

I've moved the Verilator top over to using `ibex_register_file_ff` in https://github.com/lowRISC/ibex-demo-system/pull/93, which it should have been using anyway and with which this waiver isn't needed.

Closing this PR in favour of https://github.com/lowRISC/ibex-demo-system/pull/93

**Pull request closed without merging**